### ISSUE TITLE
Trying to get in touch regarding a security issue

### ISSUE THREAD

**Issue by JamieSlome** [ethjs/ethjs-util#13]
Hey there!

I'd like to report a security issue but cannot find contact instructions on your repository.

If not a hassle, might you kindly add a `SECURITY.md` file with an email, or another contact method? GitHub [recommends](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository) this best practice to ensure security issues are responsibly disclosed, and it would serve as a simple instruction for security researchers in the future.

Thank you for your consideration, and I look forward to hearing from you!

(cc @huntr-helper)

**Comment by paulmillr** [comment]
@JamieSlome you should do full disclosure, the time has passed

**Comment by JamieSlome** [comment]
@psmoros (cc)

**Comment by paulmillr** [comment]
Last commit was 5 years ago, I doubt someone will reply.

**Comment by psmoros** [comment]
I can't find Nick's email anywhere... If you find it feel free to privately reach out otherwise you can just as well open a public issue

**Comment by paulmillr** [comment]
I would really say go full disclosure, ethjs-util is used all over the ecosystem - and keeping the vuln closed endangers users

**Comment by psmoros** [comment]
Agreed! Sorry I thought you were the original author of the report. We will triage it internally and take a decision soon :)
